test/unit/merge.js

```diff
@@ -267,20 +267,17 @@ describe('Merge', function () {
         this.merge.addFileAsync('local', _.cloneDeep(offUpdate))
         )
 
+        const doc = builders
```

Comment:
Since `doc` is the de-facto name in almost all `Merge` methods, but this `doc` isn't actually the one passed to `Merge`, won't this be a bit confusing?

What about `expectedDoc`? Or `mergedDoc`? Or maybe inline it in the expectation?